src/main/java/Event.java

```diff
@@ -0,0 +1,24 @@
+public class Event extends Task {
+    protected static final String OPEN_CURVE_BRACKET = " (";
+    protected static final String CLOSE_CURVE_BRACKET = ")";
+    protected static final String EVENT_ICON = "E";
+    protected static final String FROM_DESC = "from: ";
+    protected static final String TO_DESC = " to: ";
+
+    protected String from;
+    protected String to;
+
+    public Event(String taskName, String from, String to) {
+        super(taskName);
+        this.from = from;
+        this.to = to;
+    }
+
+    @Override
+    public String toString() {
+        return OPEN_SQUARE_BRACKET + EVENT_ICON + CLOSE_SQUARE_BRACKET
+                + super.toString() + OPEN_CURVE_BRACKET + FROM_DESC
+                + from + TO_DESC + to + CLOSE_CURVE_BRACKET;
+    }
```

Comment:
Similar to the other ones, you can consider writing out the actual string instead, as using these large amount of constants can affect the readability of the code

Reply:
Do not need to declare additional variables for strings, can directly print instead.